docs/git-commands.md

```diff
@@ -0,0 +1,147 @@
+# Git Command Guide
+The following is a number of scenarios that you will encounter while contributing to this repo. A convention in this document is to use `<< >>` to indicate text that you will need to replace when running the commands on your own machine.
+
+## Clone this REPO
+Before you begin, you will need to clone this repo
+```
+git clone https://github.com/MountSionCBSSecondary/SchoolWebsite.git
+```
+The above command will download a copy of the repo to your local machine. This copy of the repo will have the remote name of `origin`.
+
+
+## Add your own Remote
+Once you have the origin remote you will need to add a remote to your own fork of the repo. This can be done with the following command, just replace the placeholder with your own github name.
+A remote is a URL and it is github's way of specifying where a repo is stored. There is also two types of remotes `https` and `ssh` For example the school website `https` remote is `https://github.com/MountSionCBSSecondary/SchoolWebsite.git` and the `ssh` remote is `git@github.com:MountSionCBSSecondary/SchoolWebsite.git` 
+
+It should also be noted you will need to have forked the repo, how to do this can be seen in [our tutorial](Tutorial.md)
+
+```
+git remote add <<your-name>> https://github.com/<<Your Github UserName Here>>/SchoolWebsite.git
+```
+
+## Committing and Creating PR's
+So now you have following completed:
+- The repo cloned
+- Your fork remote added
+- The main repo remote added
+
+Before you can begin working on the repo we need to explain what a `branch` is. `Master` is a `branch` the main `branch` where our code will live. When we want to add features or experiment with the website we will need to create a new `branch`, when you create a new `branch` you are creating a new environment (which is identical to `master` branch) to which you can apply changes too and test without the worry of breaking, causing errors or adding bugs to our `Master` branch.
+
+When you are happy with the changes you made to your `branch` and you want to add the changes to our `Master` branch then you will need to create a `Pull Request`, this is basically where you can show the changes you made to the whole team, and as a team we can discuss and review your changes. When we are happy with the changes before we merge to `Master`.
```

Comment:
When we are happy with the changes before we merge to `Master`.  => When at least two team members have reviewed and approved the changes, the creator of the pr can then merge it to `Master`. You can also request another team member to merge you pr to master if you like.